controllers/openstackcontrolplane_controller.go

```diff
@@ -121,7 +146,7 @@ func (r *OpenStackControlPlaneReconciler) Reconcile(ctx context.Context, req ctr
 			_ = r.setProvisioningStatus(instance, newProvStatus)
 			return ctrl.Result{}, err
 		}
-		secretHash, op, err = common.CreateOrUpdateSecret(r, instance, deploymentSecret)
+		_, op, err = common.CreateOrUpdateSecret(r, instance, deploymentSecret)
```

Comment:
secretHash us used in L140, new L165, should we remove adding it to envVars ?

Reply:
This is unrelated to this patch. I can remove this change perhaps. The reason this line is changed is I already tried removing adding it to envVars and it causes a variable reference error on compile. Perhaps we can follow up on a fix for this

Reply:
Just pushed a version to drop this change...